db/schema.rb

```diff
@@ -215,7 +215,7 @@
     t.string   "cardholder_name"
     t.string   "card_debit"
     t.string   "card_last_4"
-    t.string   "card_vault_token"
+    t.string   "default_payment_method_id"
```

Comment:
Yeah the migration is gone. I think I have a mounted volume issue. Let me investigate.